packages/wdio-config/src/node/ConfigParser.ts

```diff
@@ -31,7 +31,9 @@ interface TestrunnerOptionsWithParameters extends Omit<Options.Testrunner, 'capa
 
 interface MergeConfig extends Omit<Partial<TestrunnerOptionsWithParameters>, 'specs' | 'exclude'> {
     specs?: Spec[]
+    'wdio:specs'?: Spec[]
     exclude?: string[]
+    'wdio:exclude'?: string[]
```

Comment:
Can we revert changes in this file? These are options and not capabilities. We don't need to support prefixed options as we don't have to adhere to a standardized interface

Reply:
kk, done